### PR TITLE
labs page Search's Option font Update

### PR DESCRIPTION
**Pull request by ArshdeepGrover**

### CODE REVIEW

**Comment by ArshdeepGrover** [comment]
Update: Search Option, Search bar placeholder font size
Update: Create button font size
Screenshot from 2022-06-03 09-58-30
Screenshot from 2022-06-03 10-01-11
Screenshot from 2022-06-03 09-58-33

**Comment by ArshdeepGrover** [comment]
Update: Card main heading Font size
Screenshot from 2022-06-03 10-02-27

**Comment by ArshdeepGrover** [comment]
Update: Header font updated 
Screenshot from 2022-06-03 10-55-29
Screenshot from 2022-06-03 10-55-44

**Comment by ArshdeepGrover** [comment]
update: Card font update 
Screenshot from 2022-06-03 18-30-25

File: projects/commudle-admin/src/app/feature-modules/labs/components/labs/labs.component.scss [hunk 1]
```diff
@@ -4,9 +4,8 @@
   .labs-popular-tags {
     .labs-popular-tag {
       color: $gunmetal;
-      font-size: $fs-h3;
       font-weight: $fw-400;
-      text-transform: lowercase;
+      @apply com-text-xl com-capitalize;
```
Comment (arpansac) [inline projects/commudle-admin/src/app/feature-modules/labs/components/labs/labs.component.scss:8]: Why have we changed from lowercase to capitalize?

**Review by arpansac — CHANGES_REQUESTED** [review]

File: projects/commudle-admin/src/app/feature-modules/labs/components/labs/labs.component.scss (hunk 1 above)
Comment (ArshdeepGrover) [reply projects/commudle-admin/src/app/feature-modules/labs/components/labs/labs.component.scss:8]: lowercase does not look good that's why!

**Pull request merged**